gbtis/AdminWindow.xaml.cs

```diff
@@ -1,137 +1,94 @@
 ﻿using Microsoft.Kinect;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows;
-using System.Windows.Controls;
-using System.Windows.Data;
-using System.Windows.Documents;
-using System.Windows.Input;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
 using System.Windows.Media;
-using System.Windows.Media.Imaging;
-using System.Windows.Shapes;
-
-namespace gbtis {
-    /// <summary>
-    /// Interaction logic for AdminWindow.xaml
-    /// </summary>
-    public partial class AdminWindow : Window {
-
-        public event EventHandler Exit;
-        public event EventHandler About;
-        public event EventHandler Standby;
-
-        /// <summary>
-        /// Initialize the window
-        /// </summary>
-        public AdminWindow() {
-            InitializeComponent();
-        }
-
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace gbtis {
+    /// <summary>
+    /// Interaction logic for AdminWindow.xaml
+    /// </summary>
+    public partial class AdminWindow : Window {
+
+        // Window events
+        public event EventHandler Exit;
+        public event EventHandler Standby;
+        public event EventHandler Input;
+
+        /// <summary>
+        /// Initialize the window
+        /// </summary>
+        public AdminWindow() {
+            DataContext = this;
+            InitializeComponent();
+        }
+
         /// <summary>
         /// Set the status bar's text
         /// </summary>
-        /// <param name="text"></param>
+        /// <param name="text"></param>
         public void SetStatus(String text) {
             statusText.Text = text;
-        }
-
+        }
+
         /// <summary>
         /// Update the frame in the video feed
         /// </summary>
-        /// <param name="frame">The new frame</param>
+        /// <param name="frame">The new frame</param>
         public void FrameArrived(ImageSource frame) {
             sensorFeed.Source = frame;

```

Comment:
I know it's not part of the PR but doesn't this need to be wrapped in a dispatcher?